ephios/core/templates/core/userprofile_list.html

```diff
@@ -1,64 +1,101 @@
+                        <div class="row">
+                            <div class="col col-lg-6 col-xxl-8">
+                                <h5 class="mb-0 d-inline">
+                                    {{ userprofile.get_full_name }}
+                                </h5>
+                                <br/>
+                                <a class="link-dark link-underline-opacity-0 link-opacity-50 link-underline-opacity-50-hover link-opacity-100-hover"
+                                   href="mailto:{{ userprofile.email }}">{{ userprofile.email }}</a>
+                            </div>
+                            <div class="col-auto col-lg-6 col-xxl-4">
+                                {% if userprofile.is_minor %}
+                                    <span class="badge bg-warning">{% translate "Minor" %}</span>
+                                {% endif %}
+                                {% if not userprofile.is_active %}
+                                    <span class="badge bg-danger">{% translate "Inactive" %}</span>
+                                {% elif userprofile.is_staff %}
+                                    <span class="badge bg-info">{% translate "Admin" %}</span>
+                                {% endif %}
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-8 col-lg">
+                        <div>
+                            {% if userprofile.groups.all %}
+                                <span class="fas fa-users"></span>
+                                {{ userprofile.groups.all|join:", " }}
+                            {% else %}
+                                <span class="fas fa-users-slash" title="{% translate "no groups" %}"></span>
+                            {% endif %}
+                        </div>
+                        <div class="text-body-secondary">
+                            {% for abbreviation in userprofile.qualification_grants.all|grants_to_essential_abbreviations %}
+                                <span class='badge bg-secondary text-dark'>{{ abbreviation }}</span>
+                            {% endfor %}
+                        </div>
+                    </div>
+                    <div class="col-4 col-lg-auto d-flex align-items-center justify-content-end">
                         <a class="btn btn-secondary btn-sm text-nowrap"
-                               href="{% url "core:workinghours_detail" userprofile.pk %}"><span
-                                    class="fa fa-eye"></span> <span
-                                    class="d-none d-md-inline">{% trans "View working hours" %}</span></a>
+                           href="{% url "core:workinghours_detail" userprofile.pk %}"><span
+                                class="fa fa-eye"></span> <span
+                                class="d-none d-md-inline">{% translate "Working hours" %}</span></a>
```

Comment:
maybe we should switch the icon here, as I accidentally clicked on this button because I assumed that the eye would take me to the R view

Reply:
I agree.